### PR TITLE
Update filters to include Police Station one is detained at

### PR DESCRIPTION
**Pull request by cnagadya**
Styles are likely off. Didnt get a chance to look into them. Also did most of the testing in the console. Feel free to extensively test before merging 😅

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -65,16 +65,22 @@ <h1>Kidnappings & Missing Persons in Uganda</h1>
...
+                <option value="kidnapped">Kidnapped</option>
+                <option value="found">Found</option>
+            </select>
+            <label for="detained-at">Detained At: </label>
+            <select name="detained-at" id="detained-at">
+                <option value="all">All</option>
+            </select>
 			<input type="text" id="searchInput" placeholder="Search Person's Name...">
 		</div>
+        <button id="clear-filters">Clear Filters</button>
```
Comment (Slam256) [inline index.html:83]: Screenshot 2024-07-23 at 10 55 29 AM
The text is conflicting with the background color, please change the color to maybe black.

**Review by Slam256 — CHANGES_REQUESTED** [review]

File: js/scripts.js [hunk 2]
```diff
@@ -90,43 +92,73 @@ document.addEventListener("DOMContentLoaded", function() {
...
+                card.style.display = 'none';
             }
-        }
+        })
+    }
+
+    // Filter by detaining police stations
+
+    const selectedPoliceStation = document.getElementById('police-station');
+    selectedPoliceStation.addEventListener('change', (event) => {
```
Comment (Slam256) [inline js/scripts.js:114]: element with id police-station doesn't exist

**Review by Slam256 — CHANGES_REQUESTED** [review]
- Removing the data-categories in the html and then referencing them in the script file will cause this application to not work as expected. You should put them back

**Comment by Musacoli** [comment]
@cnagadya ping here to resolve conflicts and also address the comments

File: js/scripts.js (hunk 2 above)
Comment (cnagadya) [reply js/scripts.js:114]: that should be detained_at as per this https://github.com/wkambale/missingpersons/pull/45/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R78 🤦🏽‍♀️

**Comment by cnagadya** [comment]
> * Removing the data-categories in the html and then referencing them in the script file will cause this application to not work as expected. You should put them back

That was an oversight, indeed `data-category`s should be added to the Category options

**Comment by cnagadya** [comment]
Will reopen a new PR

**Pull request closed without merging**